### DISCUSSION TITLE
I can easily add motion sensors

### DISCUSSION THREAD

**Discussion by Jessel723** [keatontaylor/alexa-actions#225] [Ideas]
I want to use this to allow Alexa to know where I'm at and send a notification there I love the idea but I don't see an option to add a device that goes that will trigger a question I don't know how to do that because I'm a noob and it would make sense to be able to use multiple things from the UI like other notify blueprint it will tell you when an alliance is done. It will tell you if plex or pfsensense or unifi goes down what makes yours unique is the motion I absolutely love the idea and I appreciate the hard work. But I would be greatly appreciative if this was a one-stop smart notification system that's smart and knows where to send the notification.  I have another that just pushes it to all of them.

**Comment by DEADSEC-SECURITY** [comment]
Nop, not with current HA limitations. The only way is to create a script for each person and than have that script look what alexa device is closer to the user (I used a BLE Beacon for this) and than run the message there.

I do have a blueprint that might make it easier.

![image](https://github.com/keatontaylor/alexa-actions/assets/45077772/edf0a4cb-d94d-45ff-a4bf-5f28069e7eb2)


Basically I have these scripts:
The one called "Talk to someone" gets a beacon id and looks which alexa device is closer to it
The one called "Talk to <person>" basically has the beacon of each user
And then the alexa talk script.

I can share these if your interested but you will need to get some ESP32 to create the beacons
